ioc_copier/ioc_copier.py

```diff
@@ -0,0 +1,104 @@
+from shutil import copytree
+from pathlib import Path
+import os
+import re
+ioc = "DFKPS"  # The name of the ioc to run, folders to copy will be of the form, <ioc>-IOC-01App and ioc<ioc>-IOC-01
+start_copy = 11  # The folder to copy (avoid IOC1 as other IOCs usually reference this rather than copying it directly)
+initial_copy = 12  # The first copy
+max_copy = 35  # The final copy
+
+answer = ""
+while answer != "y":
+    answer = input("Have you copied ioc copier to EPICS/ioc/master/<ioc_to_copy>? y/n: ")
+    if answer == "n":
+        print("please make a copy of the file at that location, then run that file.")
+        exit()
+    else:
+        if answer != "y":
+            print("please answer y or n")
+answer = ""
+while answer != "y":
+    answer = input("Have you set the ioc, start_copy, initial_copy, and max_copy variables in the file? y/n: ")
+    if answer == "n":
+        print("open the file and set those variable to match the ioc to copy")
+        exit()
+    else:
+        if answer != "y":
+            print("please answer y or n")
```

Comment:
Could this be placed into a function so we can possibly add a test in the future if we ever feel the need too while also making the functionality more reusable by following the DRY methodology?